boards/silabs/radio_boards/siwx917_rb4338a/siwx917_rb4338a_defconfig

```diff
@@ -2,8 +2,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=180000000
+# WiseConnect create threads with realtime priority. Default (10kHz) clock tick
+# prevent proper use of the system with these threads.
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=100
+
+# FIXME: A random generator is required for CONFIG_NETWORK. Replace it with a
+# real random generator.
+CONFIG_TEST_RANDOM_GENERATOR=y
+
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_PINCTRL=y
+CONFIG_NETWORKING=y
+CONFIG_WIFI=y
```

Comment:
This is the kind of stuff that really isn't (IMO) core functionality that should always be enabled. Putting this here means that you can't actually build an application without these features even though the application doesn't need them. A better place would be `Kconfig.defconfig` if you want to change the defaults, but really I think these should just come from the application. Are there some applications in Zephyr expecting these but not enabling the options from the app?

Reply:
Also, so we still get CI coverage in our own tree even when these options aren't always enabled, let's add some WiFi app to our workflows.

Reply:
Ack.